specification/datalake-analytics/resource-manager/Microsoft.DataLakeAnalytics/stable/2016-11-01/account.json

```diff
@@ -242,6 +242,15 @@
             "schema": {
               "$ref": "#/definitions/DataLakeAnalyticsAccount"
             }
+          },
+          "409": {
```

Comment:
>409 [](start = 11, length = 3)

This isn't the complete list of the responses you can return. What about 500, 503, 504, 499? Also, the error response is not RPC-compliant. (Take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content) for details.)

Instead of listing every possible failure, you should simply declare a "default" response as a catch-all for all errors with a response that matches the link I provided above.

Reply:
The comment applies to all similar changes below.

---
In reply to: [513045359](https://github.com/Azure/azure-rest-api-specs/pull/11289#discussion_r513045359) [](ancestors = 513045359)

Reply:
Added the default response for all paths and responses

Reply:
Did you push the commit? The last commit I'm seeing is from ~8 hours ago.

Reply:
yes pushed